.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1 +1,11 @@
 blank_issues_enabled: false
+contact_links:
```

Comment:
This is great!

for other reviewers: https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser